docs/product/performance/requests.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Requests
+sidebar_order: 55
+description: "Hello"
+---
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+If you have performance monitoring enabled, you can track your application's HTTP requests in Sentry. Sentry monitors all outgoing HTTP requests and helps you track and investigate the domains that serve those requests.
+
+On the [**Requests** page](#requests-page), you get an overview of the domains that serve your application's outgoing requests. From there, you can check each individual domain's behavior on its [**Domain Summary** page](#domain-summary-page) and find sample events in the [Sample List](#sample-list).
+
+<div style={{"height":"0px","paddingBottom":"calc(56.8359% + 41px)","position":"relative","width":"100%"}}>
+  <iframe
+    src="https://demo.arcade.software/BACnYWFHTvhZU6YK1rpq?embed&show_copy_link=true"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style={{"colorScheme":"light","height":"100%","left":"0px","position":"absolute","top":"0px","width":"100%"}}
+    title="Performance -> Requests Module Walkthrough"
+  ></iframe>
+</div>
+
+## Eligibility
+
+Availability of HTTP request monitoring depends on the SDK your application uses. In most cases, Sentry's SDKs automatically enable HTTP request tracking. You can check your SDK's <PlatformLink to="/performance/instrumentation/automatic-instrumentation">automatic instrumentation documentation</PlatformLink> to see if it includes tracking HTTP requests.
```

Comment:
should we cover span eligibility requirements similar to what we have for db spans? for custom instrumentation?

Reply:
Ah, good idea! I'll do that in a follow-up PR 👍🏻